cypress/support/commands.js

```diff
@@ -24,10 +24,10 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
 
-import firebase from 'firebase/app';
-import 'firebase/auth';
-import 'firebase/database';
-import 'firebase/firestore';
+import firebase from 'firebase/compat/app';
```

Comment:
Thanks so much Ben for updating our firebase code to the modular Web SDK! Is there any reason we still need to use the version 8 SDK through `compat` for cypress?

Reply:
That's what's suggested by the readme for [cypress-firebase](https://github.com/prescottprue/cypress-firebase) (look for "SDK version 9"). I imagine this is so they don't have major discrepancies between their versions, and their implementation can keep using the version 8 SDK.